src/js/components/form-elements/TableBodyVirtualized.jsx

```diff
@@ -88,6 +99,16 @@ class TableBodyVirtualized extends Component {
     const { totalCount } = properties;
 
     if (fields.value[index]) {
+      const dynamicRowAttr = fieldsConfig.getDynamicRowAttr ?
+        fieldsConfig.getDynamicRowAttr({
+          ...properties,
+          rowValues: _.filter(fields.value, v => !v.hideRow)[index],
```

Comment:
Why are you filtering the fields here?

Reply:
I think this is something, that I forgot to remove. I'll check it.